pages/nft/iscn/_iscnId.vue

```diff
@@ -162,8 +181,9 @@ export default class NFTTestMintPage extends Vue {
   }
 
   get buttonText(): string {
-    if (this.state === 'done') return this.isWritingNFT ? 'Go to Doc' : 'View NFT'
-    if (this.state === 'mint') return 'Mint NFT'
+    if (this.errorType) return 'Retry'
+    if (!this.errorType && this.state === 'done') return this.isWritingNFT ? 'Go to Doc' : 'View NFT'
+    if (!this.errorType && this.state === 'mint') return 'Mint NFT'
```

Comment:
we don't need `!this.errorType` here since we have `if (this.errorType)` in front?